libs/design/src/molecules/qty-dropdown/qty-dropdown.component.spec.ts

```diff
@@ -21,7 +22,8 @@ describe('DaffQtyDropdownComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [
-        FormsModule
+        FormsModule,
+        DaffSelectModule
```

Comment:
DaffInputModule should be imported here too

Reply:
fixed in 874a0a3